projects/angular/src/layout/breadcrumbs/breadcrumbs.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2016-2024 Broadcom. All Rights Reserved.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+
+import { ClrCommonStringsService } from '../../utils/i18n/common-strings.service';
+import { MAX_DISPLAY_ITEMS } from './breadcrumbs.constants';
+import { BreadcrumbItem } from './model/breadcrumbs.model';
+
+@Component({
+  selector: 'clr-breadcrumbs',
+  templateUrl: './breadcrumbs.html',
+  styleUrls: ['./_breadcrumbs.clarity.scss'],
+  host: {
+    class: 'clr-breadcrumb',
+    '[attr.aria-label]': 'commonStrings.keys.breadcrumb',
+    '[attr.role]': '"navigation"',
+  },
+})
+export class ClrBreadcrumbs {
+  isExpanded = false;
+  max: number = MAX_DISPLAY_ITEMS;
+  limit: number = MAX_DISPLAY_ITEMS;
```

Comment:
```suggestion
  protected max: number = MAX_DISPLAY_ITEMS;
  protected limit: number = MAX_DISPLAY_ITEMS;
```

Reply:
Also, why do we have both `max` and limit`? Could this implemented with just one max/limit property?

Reply:
`max` is used to control when to show/hide the ellipses (this will not change), `limit` is used to filter the number of breadcrumbs displayed on the screen (initially last 3) , when the breadcrumbs are expanded limit will change to the total number of breadcrumbs.